File: greencab/src/main/java/fr/pantheonsorbonne/ufr27/miage/model/BankAccount.java
package fr.pantheonsorbonne.ufr27.miage.model;
import javax.persistence.*;

@Table(name = "Bank_Account")
@Entity
public class BankAccount {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "account_id", nullable = false)
    private Integer id;

    @Column(name = "iban", nullable = false, length = 50)
    private String iban;

    @Column(name = "bic", length = 15)
    private String bic;

    public String getBic() {
        return bic;
    }

    public void setBic(String bic) {
        this.bic = bic;
    }

    public String getIban() {
        return iban;
    }

    public void setIban(String iban) {
        this.iban = iban;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }
}

File: greencab/src/main/portfolio/src/model/BankAccount.ts

export class BankAccount{
  private _IBAN: string;
  private _BIC: string;

  constructor(IBAN: string, BIC: string) {
    this._IBAN = IBAN;
    this._BIC = BIC;
  }

  get IBAN(): string {
    return this._IBAN;
  }

  set IBAN(value: string) {
    this._IBAN = value;
  }

  get BIC(): string {
    return this._BIC;
  }

  set BIC(value: string) {
    this._BIC = value;
  }
}
